package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
What is the value of adding this file in this pull request? Does it serve the task?

Reply:
I didn't add it. Wasn't it ignored?

Reply:
You did add it: see the [diff](https://github.com/PureBhaktiArchive/audioseva/pull/50/commits/919c043d92203c848c1c56cbb8b7d4c2ad654f27#diff-32607347f8126e6534ebc7ebaec4853d) yourself.